concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+### Sprint Planning
+With our backlog and sprint boards established, we can flesh out our first meeting: sprint planning.
+
+This is meant to be a time to consider what work the team wants to accomplish in the next two-week sprint and assign relevant tasks to people. It should happen on Mondays and take about an hour, following the sprint retrospective (detailed later).
+
+It should start with a brief discussion about the team’s goals. This should be an open, but high-level, discussion that will help frame the kinds of issues we should prioritize. It shouldn’t take long -- no more than 15-20 minutes.
+
+With the context of that discussion in mind, we’ll move to backlog review. As detailed above, our backlogs are contained in individual boards, and we’ll run through each one as part of the planning process.
+
+For each board:
+- Briefly run through the Backlog column to decide if there are any issues we want to promote to "Tracking" status.
+- Run through all items in "Tracking" and decide which ones are highest priority, and who will work on them. They will become the issue’s assignee.
+
+From there, automation will aid the process -- any newly assigned issues will:
+- be moved to the Active column in the backlog board
+- automatically added to the Assigned column of the Active Sprint board
+
+This process repeats for every board, after which our Active Sprint board will be populated with our workload for the next two weeks, and our sprint planning is complete.
+
+### Daily Standup
+Our daily standups will take place every day we don’t have a longer meeting (such as sprint planning/retro) for half an hour. This is meant to be a quick check-in to ensure newly-opened issues get quick responses and blockers get resolved.
+
+First, if there are any new people on the call, we'll welcome them and allow them to ask questions or have discussions as needed.
+
+Next, we’ll have a look at all untriaged issues -- any that do not have a label, and are not part of a project. This will allow the whole team to discuss them and ensure they get moved to the appropriate backlog board and labeled correctly. This shouldn’t take more than 15 minutes, though that’s a soft limit -- if a good conversation is happening, it shouldn’t be cut off.
+
+Finally, if there's time, we'll dedicate some time to a quick review of our Current Sprint board. This will serve as an opportunity for developers to elaborate on any blockers and ask for reviews or re-reviews as needed. We should strive to do this every day if possible, but if prior discussion makes that infeasible, we don't _have_ to do it (there's always tomorrow). Project discussion is more important.
```

Comment:
Maybe it's just me, but it seems like a daily discussion of blockers / opportunity to ask questions and report progress is the biggest value of our current stand-up. Do we actively want this to change? If so, that could be fine (we still have the chat for that sort of thing) but I just wanted to point out that this description seems very different from what we're doing currently.

Reply:
Agree - I'd expect individual reports to happen before untriaged issues discussion.

Reply:
What I was trying to express here was that those discussions around blockers (which are valuable, absolutely) should happen in the context of the issues themselves in the current sprint as opposed to just a quick status update.

If the current sprint board has everything we're working on anyway, we'd be duplicating status by talking about it in an individual report and then later on when we're looking at the board. That's redundant and a potential waste of time, IMO.